.github/workflows/test.yml

```
@@ -244,8 +244,9 @@ jobs:
       uses: codecov/codecov-action@v4
       continue-on-error: true
       with:
-        fail_ci_if_error: true
+        fail_ci_if_error: false
         files: ./coverage.xml
+        token: ${{ secrets.CODECOV_TOKEN }}
```

```suggestion
        token: 1eca3b1f-31a2-4fb8-a8c3-138b441b50a7 #repo token
```

might want to put it into the config